### PR TITLE
Submits Ticket 5. View a separate page for each category with a list …

### PR DESCRIPTION
**Pull request by stef-lukes**
…of related reviews

- Adds 2 Routes to the reviews page in App.js, one with parametric endpoint for categories
- Adds Navbar component to App.js
- Adds useParams to ReviewsList which watches for 'category' when fetching reviews
- Adds a getCategories function to api.js
- Adds a DropdwonMenu component for use in navbar re: categories, which dynamically creates category links in state

### CODE REVIEW

File: src/components/DropdownMenu.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import { useState } from "react";
+import { getCategories } from "../utils/api";
+import { useParams } from "react-router-dom";
+
+const DropdownMenu = () => {
+  const { category } = useParams();
+  const [categories, setCategories] = useState([]);
+
+  getCategories(category).then((categories) => {
+    console.log(categories);
```
Comment (mallionaire) [inline src/components/DropdownMenu.jsx:11]: remove console.log

File: src/components/DropdownMenu.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { Link } from "react-router-dom";
+import { useState } from "react";
+import { getCategories } from "../utils/api";
+import { useParams } from "react-router-dom";
+
+const DropdownMenu = () => {
+  const { category } = useParams();
+  const [categories, setCategories] = useState([]);
+
+  getCategories(category).then((categories) => {
```
Comment (mallionaire) [inline src/components/DropdownMenu.jsx:10]: if you take a look at this in console in the browser you have an infinite loop running, how can you contain this getCategories logic to ensure it only fetched the data once?

File: src/App.js [hunk 3]
```diff
@@ -2,6 +2,8 @@ import "./App.css";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
 import Header from "./components/Header";
 import Home from "./pages/Home";
+import Reviews from "./pages/Reviews";
+import Navbar from "./components/Navbar";
 // import NavBar from "./components/NavBar";
```
Comment (mallionaire) [inline src/App.js:7]: remove commented out code

File: src/pages/Reviews.jsx [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import ReviewsList from "../components/ReviewsList";
+
+const Reviews = () => {
+  return (
+    <>
```
Comment (mallionaire) [inline src/pages/Reviews.jsx:5]: do you need a fragment here?

**Review by mallionaire — CHANGES_REQUESTED** [review]
Bit of an infinite loop happening here... 
Think about how we can manage that 😄

**Comment by scar1377** [comment]
OK to merge

**Pull request merged**